minigalaxy/filesys_utils.py

```diff
@@ -0,0 +1,174 @@
+import subprocess
+import os
+import shutil
+import json
+from minigalaxy.paths import CONFIG_DIR, CACHE_DIR, CONFIG_FILE_PATH, DEFAULT_INSTALL_DIR
+
+
+def _get_white_list():
+    json_config = {"install_dir": DEFAULT_INSTALL_DIR}
+    if os.path.exists(CONFIG_FILE_PATH):
+        with open(CONFIG_FILE_PATH, "r") as file:
+            try:
+                json_config = json.loads(file.read())
+            except json.decoder.JSONDecodeError:
+                pass
+    return [json_config["install_dir"], CONFIG_DIR, CACHE_DIR]
+
+
+def _get_black_list():
+    important_user_dirs = ["", "DESKTOP", "DOCUMENTS", "DOWNLOAD", "MUSIC", "PICTURES", "PUBLICSHARE", "TEMPLATES",
```

Comment:
This seems a bit specific. Could you explain the logic behind this? I don't think we are removing directories which are filled with files other than installation directories at the moment.

Reply:
Those are standard xdg-user-dir
User technically can remove them, but we for sure don't want to do this. This would be the most damaging thing that can happen by accident.

Those entries get localized to actual user directories. Lets take "VIDEOS" for example:
```
>>> import subprocess
>>> subprocess.check_output(['xdg-user-dir', "VIDEOS"]).decode("utf-8").strip()
'/home/makson/Wideo'
>>> 
```